packages/client-api-schema/schema/close-channel.json

```diff
@@ -0,0 +1,52 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://statechannels.org/schemas/close-channel.json",
+  "definitions": {
+    "CloseChannelResult": {
+      "type": "object",
+      "properties": {
+        "participants": {
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/definitions/Participant"
+          },
+          "title": "participants"
+        },
+        "allocations": {
+          "type": "array",
+          "items": {
+            "$ref": "definitions.json#/definitions/Allocation"
+          },
+          "title": "allocations"
+        },
+        "appDefinition": {
+          "$ref": "definitions.json#/definitions/Address",
+          "title": "appDefinition"
+        },
+        "appData": {
+          "type": "string",
+          "title": "appData"
+        },
+        "status": {
+          "type": "string",
+          "const": "opening",
+          "enum": ["proposed", "opening", "funding", "running", "closing", "closed"]
+        },
+        "turnNum": {
+          "type": "string",
+          "const": "0x00"
+        }
```

Comment:
This set of attributes (elsewhere known as a `ChannelResult`) appears together in virtually all the responses. It therefore might be useful to introduce a new schema definition for this type, which help identify it a key concept.

That would allow us to flip some of this work on its head - reintroducing the `ChannelResult` type, but with that type defined automatically from the schema.